WhaleSpotting/Controllers/UserController.cs

```diff
@@ -40,4 +42,18 @@ public IActionResult Create([FromBody] UserRequest newUser)
         var url = Url.Action("GetById", new { userId = user.Id });
         return Created(url, new UserResponse(user));
     }
+
+    [HttpGet("ListUsers")]
+    public ActionResult<List<UserResponse>> ListOfUsers()
+    {
+        try
+        {
+            var usersList = _userService.ListAllUsers();
```

Comment:
We don't need to call this variable a list as it should be explicit so can just call it `users`
```suggestion
            var users = _userService.ListAllUsers();
```

Reply:
changed